src/scenes/game-scene.ts

```diff
@@ -134,15 +138,27 @@ class GameScene extends Phaser.Scene {
     toggleMusic(this); // attaches listener to mute button
     const [isMute] = useLocalStorage('isMute', false);
     this.game.sound.mute = isMute; // set game mute to saved ls value
+
+    this.player = this.level.spawners.player.getChildren()[0] as Ben3;
+
+    const canvas = document.querySelector('#game');
+    canvas?.addEventListener('click', this.jump.bind(this));
+  }
+
+  jump() {
+    if (!this.level || !this.player) return;
+    this.player.jump();
   }
 
   update() {
-    if (!this.parallax || !this.level) return;
+    if (!this.parallax || !this.level || !this.player) return;
 
     this.parallax.update();
 
-    this.player = this.level.spawners.player.getChildren()[0] as Ben1;
-    smoothMoveCameraTowards(this, this.player.torso, 0.9);
+    smoothMoveCameraTowards(this, this.player.gameObject, 0.8);
+
+    const [myNum, setMyNum] = useLocalStorage('testNum', 0);
+    setMyNum(myNum + 1);
```

Comment:
bit of Debugging aye?

Reply:
seems to be running well on every frame!
i just checked and my value is `6568819` - which is the number of frames i've rendered over lifetime of having this code here

Reply:
would be good for collected coin count